lib/callbacks.js

```diff
@@ -90,6 +90,21 @@ const unsafeFunction = fn => (typeof fn === 'function' ? fn : null);
 // Returns: <Function>, function or `common.emptiness` if fn is not a function
 const safeFunction = fn => (typeof fn === 'function' ? fn : emptiness);
 
+// Identity function
+//   x - <any>, incoming value which will be returned
+// Returns: <any>, incoming value
+const id = x => x;
+
+// Async identity function
+//   x - <any>, incoming value which will be returned in callback
+//   callback - <Function>, callback to be called with first argument
+//     err - <null>, will be alse return instead of error
```

Comment:
I can't quite get the meaning of this.

Reply:
I think the description of callback arguments can be omitted, yet types should stay.